R/git.R

```diff
@@ -135,10 +135,10 @@ github_remote_parse <- function(x) {
 
   if (grepl("^https", x)) {
     # https://github.com/hadley/devtools.git
-    re <- "github.com/(.*?)/(.*)\\.git"
+    re <- "github[^/]*/(.*?)/(.*)\\.git"
   } else if (grepl("^git", x)) {
     # git@github.com:hadley/devtools.git
-    re <- "github.com:(.*?)/(.*)\\.git"
+    re <- "github[^:]*:(.*?)/(.*)\\.git"
```

Comment:
Ah you are right about them being different, my mistake. 

If we are just ignoring everything but the `user/repo` slug they might as well be combined I think.